content/manuals/build/release-notes.md

```diff
@@ -10,6 +10,28 @@ toc_max: 2
 This page contains information about the new features, improvements, and bug
 fixes in [Docker Buildx](https://github.com/docker/buildx).
 
+## 0.28.0
+
+{{< release-date date="2025-09-03" >}}
+
+The full release notes for this release are available
+[on GitHub](https://github.com/docker/buildx/releases/tag/v0.27.0).
+
+### New
+
+- When building with Dockerfile version 1.18.0 or later, you can now use new Git URLs with query options for build context and named contexts in the `build` and `bake` command. [moby/buildkit#6183](https://github.com/moby/buildkit/releases/tag/dockerfile%2F1.18.0)
```

Comment:
Made a fix in our release notes upstream, it should be:

```suggestion
- When building with Dockerfile version 1.18.0 or later, you can now use new Git URLs with query options for build context and named contexts in the `build` and `bake` command. [dockerfile/1.18.0](https://github.com/moby/buildkit/releases/tag/dockerfile%2F1.18.0) [moby/buildkit#6183](https://github.com/moby/buildkit/pull/6183)
```

For `[dockerfile/1.18.0](https://github.com/moby/buildkit/releases/tag/dockerfile%2F1.18.0)` I think we should link to our Dockerfile release notes at https://docs.docker.com/build/buildkit/dockerfile-release-notes/ when available.